tests/phpunit/tests/option/option.php

```diff
@@ -312,4 +312,254 @@ public function data_option_autoloading() {
 			array( 'autoload_false', false, 'no' ),
 		);
 	}
+
+	/**
+	 * Test prime_options function.
```

Comment:
```suggestion
	 * Tests that prime_options() primes multiple options.
```
All method, function and test method docblocks should start with a third-person singular verb (`Tests`, not `Test`), and should ideally start with `Tests that`, with a description of the scenario that is being tested.

🔢 Applies elsewhere in the PR's tests.